site/docs/v1/tech/shared/_pre-1-37-warning.adoc

```diff
@@ -0,0 +1,3 @@
+The method to start FusionAuth underwent changes in version 1.37. The below only applies to versions 1.36 and earlier.
```

Comment:
The commands to start FusionAuth changed in version `1.37.0`. The following instructions apply to versions up to `1.36.0`.